examples/fluids/qfunctions/velocity_gradient_projection.h

```diff
@@ -19,26 +19,23 @@ CEED_QFUNCTION_HELPER int VelocityGradientProjectionRHS(void *ctx, CeedInt Q, co
   const CeedScalar(*q)[CEED_Q_VLA]         = (const CeedScalar(*)[CEED_Q_VLA])in[0];
   const CeedScalar(*Grad_q)[5][CEED_Q_VLA] = (const CeedScalar(*)[5][CEED_Q_VLA])in[1];
   const CeedScalar(*q_data)                = in[2];
-  const CeedScalar(*x)[CEED_Q_VLA]         = (const CeedScalar(*)[CEED_Q_VLA])in[3];
   CeedScalar(*v)[CEED_Q_VLA]               = (CeedScalar(*)[CEED_Q_VLA])out[0];
 
   NewtonianIdealGasContext context = (NewtonianIdealGasContext)ctx;
 
   CeedPragmaSIMD for (CeedInt i = 0; i < Q; i++) {
     const CeedScalar qi[5]  = {q[0][i], q[1][i], q[2][i], q[3][i], q[4][i]};
-    const CeedScalar x_i[3] = {x[0][i], x[1][i], x[2][i]};
     CeedScalar       wdetJ, dXdx[3][3];
     QdataUnpack_3D(Q, i, q_data, &wdetJ, dXdx);
 
-    const State s = StateFromQ(context, qi, x_i, state_var);
+    const State s = StateFromQ(context, qi, state_var);
     State       grad_s[3];
     for (CeedInt k = 0; k < 3; k++) {
-      CeedScalar dx_i[3] = {0}, dqi[5];
+      CeedScalar dqi[5];
       for (CeedInt j = 0; j < 5; j++) {
         dqi[j] = Grad_q[0][j][i] * dXdx[0][k] + Grad_q[1][j][i] * dXdx[1][k] + Grad_q[2][j][i] * dXdx[2][k];
       }
-      dx_i[k]   = 1.;
-      grad_s[k] = StateFromQ_fwd(context, s, dqi, x_i, dx_i, state_var);
+      grad_s[k] = StateFromQ_fwd(context, s, dqi, state_var);
     }
```

Comment:
Reminder to myself: This probably should use `StatePhysicalGradient*` if possible.